test/classes/vass/nested-class-with-user-defined-constructor-1.bad

```diff
@@ -1 +1,9 @@
 nested-class-with-user-defined-constructor-1.chpl:4: error: constructor for class 'inner' requires a generic argument called 'outer'
+nested-class-with-user-defined-constructor-1.chpl:4: In initializer 'inner':
+nested-class-with-user-defined-constructor-1.chpl:4: internal error: EXPnnnn chpl Version mmmm
+
+Internal errors indicate a bug in the Chapel compiler ("It's us, not you"),
+and we're sorry for the hassle.  We would appreciate your reporting this bug -- 
+please see http://chapel.cray.com/bugs.html for instructions.  In the meantime,
+the filename + line number above may be useful in working around the issue.
```

Comment:
It is surprising to me that this .bad file changed to an internal error. Could you investigate more and/or explain why this isn't indicative of a greater problem? I.e. are other tests/user codes now going to run into an internal error?